### ISSUE TITLE
The bot in it's current newest build is bugging out with EXP/HR and XP Count

### ISSUE THREAD

**Issue by Lynical** [Sen66/PokemonGo-Bot#53]
Title says it pretty much ,it changes from 290k to 306k and then repeats every milli second.

**Comment by Sen66** [comment]
wut?

**Comment by Lynical** [comment]
What wut I told you what happens, where it says: Level XX - (190000/300k)
the 190000 amount jumps from 190000 to 201k for example and keeps repeating, so back to 190k after and then again to 201k and so on.

**Comment by Sen66** [comment]
eh first you said xp/hour now you talk about the xp needed for the next level. I look into it now. Thanks for notifying me.

**Comment by BosnaZmaj** [comment]
In my case it does not update at all. Only when you close the application and relaunch it, it would update,

**Comment by Lynical** [comment]
@Sen66 bugging out with EXP/HR and XP Count is what the title said.

XP count aka xp you got

**Comment by Sen66** [comment]
yeah the console title isnt updating at its current state. Fix coming asap.

**Comment by Lynical** [comment]
yeah and the xp/hr is glitching out too.

@sen66

**Comment by Sen66** [comment]
update out!
